### PR TITLE
Help hub

### PR DESCRIPTION
**Pull request by vickymg**

### CODE REVIEW

File: mtp_send_money/templates/send_money/prisoner-details-error-summary.html [hunk 1]
```diff
@@ -20,7 +20,12 @@ <h1 class="error-summary-heading heading-medium" id="error-summary-heading">
         <li class="non-field-error">
           {% url 'send_money:help' as help_url %}
           {% blocktrans trimmed %}
-            To find out why, refer to <a href="{{ help_url }}#not-found">help</a>
+          Please check:
```
Comment (ushkarev) [inline mtp_send_money/templates/send_money/prisoner-details-error-summary.html]: could you instead wrap each item in separate translation tags. that makes it easier to translate as it leaves out as much HTML as possible and translators only deal with plain text

File: mtp_send_money/templates/send_money/prisoner-details-error-summary.html [hunk 2]
```diff
@@ -20,7 +20,12 @@ <h1 class="error-summary-heading heading-medium" id="error-summary-heading">
...
+            <ul class="list list-bullet">
+              <li>you entered the details correctly</li>
+              <li>you're sending money to a <a href="{{ help_url }}prisons">public prison</a></li>
```
Comment (ushkarev) [inline mtp_send_money/templates/send_money/prisoner-details-error-summary.html]: can u replace `{{ help_url }}prisons` with {{ prison_list_url }}` and put `{% url 'send_money:prison_list' as prison_list_url %}` somewhere (maybe next to the other url tag if there is one)

File: mtp_send_money/templates/send_money/prisoner-details-error-summary.html (hunk 1 above)
Comment (vickymg) [inline mtp_send_money/templates/send_money/prisoner-details-error-summary.html]: done :-)

File: mtp_send_money/templates/send_money/prisoner-details-error-summary.html (hunk 2 above)
Comment (vickymg) [inline mtp_send_money/templates/send_money/prisoner-details-error-summary.html]: and also done!

**Pull request merged**